src/DecoratorManifold.jl

```diff
@@ -739,8 +768,36 @@ decorated_manifold(M::AbstractManifold) = M.manifold
 #
 Base.@propagate_inbounds function Base.getindex(
     p::AbstractArray,
-    M::AbstractDecoratorManifold,
+    M::AbstractDecoratorManifold{𝔽,<:AbstractDecoratorType},
     I::Union{Integer,Colon,AbstractVector}...,
-)
+) where {𝔽}
     return getindex(p, decorated_manifold(M), I...)
 end
+
+PARENT_FUNCTIONS = [
```

Comment:
```suggestion
DEFAULT_PARENT_FUNCTIONS = [
```
I think a more specific name would be better.